docker-compose-sse.yaml

```diff
@@ -0,0 +1,74 @@
+version: '3.4'
+
+services:
+
+  nginx:
+    image: nginx
+    restart: on-failure
+    build: nginx
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+    environment:
+      HOSTNAMES_DIR: "/hostnames"
+    volumes:
+      - "./tests/hostnames:/hostnames"
+    depends_on:
+      - minio
+
+  minio:
+    image: minio
+    restart: on-failure
+    build: .
+    environment:
+      MINIO_ROOT_USER: "12345678"
+      MINIO_ROOT_PASSWORD: "12345678"
+      MINIO_AUDIT_WEBHOOK_ENABLE_target1: "on"
+      MINIO_AUDIT_WEBHOOK_AUTH_TOKEN_target1: ""
+      MINIO_AUDIT_WEBHOOK_ENDPOINT_target1: http://cwm-worker-logger:8500/logs
+      MINIO_GATEWAY_DEPLOYMENT_ID: "docker-compose-http"
+      MINIO_KMS_AUTO_ENCRYPTION: "on"
+      MINIO_KMS_KES_ENDPOINT: "https://minio-kes:7373"
+      MINIO_KMS_KES_CERT_FILE: "/minio-kes/minio.cert"
+      MINIO_KMS_KES_KEY_FILE: "/minio-kes/minio.key"
+      MINIO_KMS_KES_CAPATH: "/minio-kes/server.cert"
+      MINIO_KMS_KES_KEY_NAME: "minio" # configured in `minio-kes/server-config.yaml`
+    volumes:
+      - "storage:/storage"
+      - "cache:/cache"
+      - "minio-kes:/minio-kes"
+    depends_on:
+      - minio-kes
+      - cwm-worker-logger
+
+  minio-kes:
+    image: minio-kes:v0.17.6
+    build: ./minio-kes
+    container_name: minio-kes
+    ports:
+      - "7373:7373"
+    volumes:
+      - "minio-kes:/minio-kes"
+
+  cwm-worker-logger:
```

Comment:
no need to include the logger and redis in this docker compose, just the minimal stack to test the encryption. Adding logger here will also require it to be updated if we make updates to the logger in the future